server/channels/api4/export.go

```diff
@@ -8,7 +8,7 @@ import (
 	"net/http"
 	"path/filepath"
 	"time"
-
```

Comment:
Almost. Please add this empty line back.

Reply:
Done! ❤️